README.md

```diff
@@ -6,11 +6,16 @@ Following the installation you should have a Kinto server available for you to u
 ## Installation
 
 ```
-$ git clone http://github.com/Kinto/kinto-heroku.git --depth=1 kinto-instance && cd kinto-instance
-$ heroku create && git push heroku master
+$ git clone http://github.com/Kinto/kinto-heroku.git kinto-instance && cd kinto-instance
```

Comment:
But you cannot keep the `--depth=1` because you'll face a swallow cloning problem. See here, for more details https://stackoverflow.com/questions/28983842/remote-rejected-shallow-update-not-allowed-after-changing-git-remote-url